plugin/pkg/scheduler/api/types.go

```diff
@@ -43,6 +43,7 @@ type PriorityPolicy struct {
 	// For the Kubernetes provided priority functions, the name is the identifier of the pre-defined priority function
 	Name string `json:"name"`
 	// The numeric multiplier for the minion scores that the priority function generates
+	// The weight should be non-zero and can be a positive or a negative integer
```

Comment:
Yeah, but just for consistency, if you can make negative preferences with a weight, you should also be able to have them directly. IMO.